src/approaches/grammar_search_invention_approach.py

```diff
@@ -360,6 +363,87 @@ def enumerate(self) -> Iterator[Tuple[Predicate, float]]:
                     yield (uff_predicate, cost)
 
 
+### Useful for debugging ###
+
+# # Replace these strings with anything you want to exclusively enumerate.
+# # Make sure to uncomment return _DebugGrammar(skip_grammar) in
+# # _create_grammar and the call to _run_debug_analysis.
+# _DEBUG_PREDICATE_STRS = [
+#     "((0:obj).wetness<=0.5)",
+#     "NOT-((0:obj).wetness<=0.5)",
+# ]
+
+# @dataclass(frozen=True, eq=False, repr=False)
+# class _DebugGrammar(_PredicateGrammar):
+#     """A grammar that omits given predicates from being enumerated.
```

Comment:
Is this really omitting, or is this docstring erroneously copied from elsewhere?